Uplift/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,174 @@
+
+            let tableViewBottomOffset: CGFloat = 17
+            let tableViewSize = CGSize(width: 388, height: 308)
+
+            titleLabel.snp.makeConstraints { make in
+                make.centerX.equalToSuperview()
+                make.width.equalTo(labelWidth)
+                make.height.equalTo(labelHeight)
+                make.top.equalToSuperview().inset(labelTopOffset)
+            }
+
+            table.snp.makeConstraints { make in
+                make.top.equalTo(titleLabel.snp.bottom).offset(tableViewBottomOffset)
+                make.size.equalTo(tableViewSize)
+                make.centerX.equalTo(titleLabel)
+            }
+        } else { // Without Table View
+            let imageTextPadding: CGFloat = 20
+
+            titleLabel.snp.makeConstraints { make in
+                make.top.equalTo(imageView.snp.bottom).offset(imageTextPadding)
+                make.width.equalTo(labelWidth)
+                make.centerX.equalToSuperview()
+            }
+        }
+    }
+
+    private func generateAttributedString(
+        for string: String,
+        color: UIColor = .primaryBlack
+    ) -> NSAttributedString {
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 3
+        style.alignment = .center
+        let attributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont._20MontserratBold!,
+            .paragraphStyle: style,
+            .foregroundColor: color
+        ]
+
+        return NSAttributedString(string: string, attributes: attributes)
+    }
+}
+
+extension OnboardingView: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: FavoriteCell.identifier, for: indexPath) as? FavoriteCell else { return UITableViewCell() }
+        let data = tableData?[indexPath.section] ?? ""
+        cell.configure(with: data, displaysClasses: showingClasses)
+        cell.selectionStyle = .none
+        return cell
+    }
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+}
+
+extension OnboardingView: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        guard let cell = tableView.cellForRow(at: indexPath) as? FavoriteCell, let data = tableData else { return }
+
+        cell.toggleSelectedView(selected: !cell.currentlySelected)
+
+        if cell.currentlySelected {
+            favorites.append(data[indexPath.section])
+        } else {
+            favorites.removeAll { $0 == data[indexPath.section] }
+        }
+
+        favoritesSelectedDelegate?(favorites)
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return tableViewCellHeight
```

Comment:
we can just return the number here instead of making it a class var if its not used anywhere else